scripts/generator/graphql_generator/csharp/SDK/iOS/iOSNativeCheckout.cs.erb

```diff
@@ -48,15 +48,14 @@ namespace <%= namespace %>.SDK.iOS {
         /// Checks if the device is capable of paying with Apple Pay and checks if the store's payment provider accepts Apple Pay
         /// </summary>
         /// <param name="paymentSettings">The Shop's payment settings</param>
+        /// <param name="callback">The callback that will deliver the result to tha caller</param>
         /// <returns>True if the device is capable of paying with Apple Pay and the store accepts payments with Apple Pay</returns>
-        public bool CanCheckout(PaymentSettings paymentSettings) {
-
+        public void CanCheckout(PaymentSettings paymentSettings, CanCheckoutWithNativePayCallback callback) {
             if (paymentSettings.supportedDigitalWallets().Contains(DigitalWallet.APPLE_PAY) &&
                 _CanCheckoutWithApplePay(SerializedPaymentNetworksFromCardBrands(paymentSettings.acceptedCardBrands()))) {
-                return true;
+                callback(true);
```

Comment:
Missing a return here or else you'll call the callback twice potentially.

Reply:
Good catch! Will fix it.

Reply:
https://github.com/Shopify/unity-buy-sdk/pull/360/commits/c24aa7bb3dfa3770e1d73741004f421ee6831fd6